pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -0,0 +1,198 @@
+package periodicbackupcontroller
+
+import (
+	"context"
+	"fmt"
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	backupv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1alpha1"
+	"github.com/openshift/cluster-etcd-operator/pkg/backuphelpers"
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/etcd_assets"
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver/featuregates"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	batchv1client "k8s.io/client-go/kubernetes/typed/batch/v1"
+	"k8s.io/klog/v2"
+	"time"
+
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/health"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/client-go/kubernetes"
+)
+
+type PeriodicBackupController struct {
+	operatorClient        v1helpers.OperatorClient
+	backupsClient         backupv1client.BackupsGetter
+	kubeClient            kubernetes.Interface
+	operatorImagePullSpec string
+	featureGateAccessor   featuregates.FeatureGateAccess
+}
+
+func NewPeriodicBackupController(
+	livenessChecker *health.MultiAlivenessChecker,
+	operatorClient v1helpers.OperatorClient,
+	backupsClient backupv1client.BackupsGetter,
+	kubeClient kubernetes.Interface,
+	eventRecorder events.Recorder,
+	operatorImagePullSpec string,
+	accessor featuregates.FeatureGateAccess) factory.Controller {
+
+	c := &PeriodicBackupController{
+		operatorClient:        operatorClient,
+		backupsClient:         backupsClient,
+		kubeClient:            kubeClient,
+		operatorImagePullSpec: operatorImagePullSpec,
+		featureGateAccessor:   accessor,
+	}
+
+	syncer := health.NewDefaultCheckingSyncWrapper(c.sync)
+	livenessChecker.Add("PeriodicBackupController", syncer)
+
+	return factory.New().ResyncEvery(1*time.Minute).
+		WithSync(syncer.Sync).ToController("PeriodicBackupController", eventRecorder.WithComponentSuffix("periodic-backup-controller"))
```

Comment:
I'm thinking we should probably have this controller get triggered via `.WithInformers()` on the following objects:
- Backups CR (to reconcile on updates to the config)
- CronJobs (to update on CronJob changes)

Otherwise it would be slow to react to spec changes.

I think we forgot this for the other backup controller but that should also probably be watching for its CR, and Jobs
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/backupcontroller/backupcontroller.go#L68-L69